### PR TITLE
Added Origami Cover Image in Readme.

### PR DESCRIPTION
**Pull request by vpn1997**
- Added Cover Image in Readme.
[[ Updated Readme ](https://github.com/vpn1997/Origami/tree/read1 ) ]

### CODE REVIEW

**Comment by AvaisP** [comment]
Is it possible to remove "CVFY code" from the cover image?

File: README.md [hunk 1]
```diff
@@ -1,20 +1,23 @@
-# Origami
-
-<p align="center"><img width="45%" height="15%" src="origami.png" /></p>
+<p align="center"><img width="30%" height="15%" src="origami_logo.png" /></p>
 
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
```
Comment (uttu357) [inline README.md]: Why?

Comment (vpn1997) [reply README.md]: center tag was not working thats why i used &nbsp.

Comment (vpn1997) [reply README.md]: https://github.com/anfederico/Clairvoyant , they  follow same styling.

**Comment by vpn1997** [comment]
@AvaisP hey the cover image was provided by @deshraj .

**Comment by vpn1997** [comment]
@uttu357  Done !

**Pull request merged**